README.md

```diff
@@ -1,14 +1,59 @@
-# CS3219-AY22-23-Project-Skeleton
+# CS3219-AY22-23-S1-G13
 
-This is a template repository for CS3219 project.
+# Introduction 
+This project is an adaptation of the project PeerPrep that has been completed in the previous semesters of CS3219. This project provides a collaborative technical interview training platform, where one can receive a coding question, and collaborate with someone to answer the question. 
+
+
+# Running Project on Deployed Service
+
+The project is currently deployed [here](https://frontend-66acladbaq-uc.a.run.app). All APIs are also deployed, and able to be accessed through the frontend from the link. 
+
+# Running Project Locally
+
+## Prerequisites before starting
+Our current repository is currently deployed, and the URLs in the frontend is configured to call the APIs that are deployed. 
+To run the project locally, change all URLs to `localhost`, and change the port number accordingly (provided below)
+
+We also assume that `npm` is available on your local device. If `npm` is not available, click [here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) 
 
 ## User Service
-1. Rename `.env.sample` file to `.env`.
-2. Create a Cloud DB URL using Mongo Atlas.
-3. Enter the DB URL created as `DB_CLOUD_URI` in `.env` file.
-4. Install npm packages using `npm i`.
-5. Run User Service using `npm run dev`.
-
-## Frontend
-1. Install npm packages using `npm i`.
-2. Run Frontend using `npm start`.
+Ensure that frontend directs to `localhost:8000` for user-service APIs. 
+
+1. Change directory to user-service by running `cd user-service`
+2. Ensure the dependencies are downloaded locally `npm install` 
+3. Start the user-service by running `npm run start` 
+
+## Collaboration Service
+Ensure that frontend directs to `localhost:8003` for collaboration-service APIs. 
+
+1. Change directory to user-service by running `cd collaboration-service`
+2. Ensure the dependencies are downloaded locally `npm install` 
+3. Start the user-service by running `npm run start` 
+
+## History Service
+Ensure that frontend directs to `localhost:8005` for history-service APIs. 
+
+1. Change directory to user-service by running `cd history-service`
+2. Ensure the dependencies are downloaded locally `npm install` 
+3. Start the user-service by running `npm run start` 
+
+## Matching Service

```

Comment:
Change the url for io connection to localhost:3000 to connect to frontend locally